data/articles/04395.json

```diff
@@ -9,7 +9,9 @@
             "type": "research-article",
             "doi": "10.7554/eLife.04395",
             "title": "Mapping the zoonotic niche of Ebola virus disease in Africa",
+            "titlePrefix": "Title prefix",
```

Comment:
Could you remove this for all of them except 10627? It's only going to be used by certain article types.